train.py

```diff
@@ -83,29 +83,19 @@ def main():
         data_range=1
     ).to(device)
 
-    if hparams.deployment:
-        model_config = {
-            'scale': hparams.scale,
-            'pos_encoder_type': 'hash',
-            'levels': 4,
-            'feature_per_level': 4,
-            'base_res': 32,
-            'max_res': 128,
-            'log2_T': 21,
-            'xyz_net_width': 16,
-            'rgb_net_width': 16,
-            'rgb_net_depth': 1,
-        }
-    else:
-        model_config = {
-            'scale': hparams.scale,
-            'pos_encoder_type': hparams.encoder_type,
-            'max_res': 1024 if hparams.scale == 0.5 else 4096,
-            'half_opt': hparams.half_opt,
-        }
+    model_config = {
+        'scale': hparams.scale,
+        'half_opt': hparams.half_opt,
+        'sh_degree': hparams.sh_degree,
+        'grid_size': hparams.grid_size,
+        'grid_radius': hparams.grid_radius,
+        'origin_sh': hparams.origin_sh,
+        'origin_sigma': hparams.origin_sigma
+    }
 
     # model
-    model = NGP(**model_config).to(device)
+    # model = NGP(**model_config).to(device)
+    model = VoxelGrid(**model_config).to(device)
```

Comment:
Hi, can you make this part configurable so it works with both NGP or Plenoxel with a flag toggle? That'd be best, thank you!

Reply:
For sure, I will add this feature soon.